packages/commons/fs-utils/src/basename.ts

```diff
@@ -0,0 +1,11 @@
+
+import path from "path";
+import { AbsoluteFilePath } from "./AbsoluteFilePath";
+import { RelativeFilePath } from "./RelativeFilePath";
+
+export function basename(filepath: RelativeFilePath): RelativeFilePath;
+export function basename(filepath: AbsoluteFilePath): AbsoluteFilePath;
+export function basename(filepath: string): string;
+export function basename(filepath: string): string {
+    return path.basename(filepath);
```

Comment:
why call this `basename` instead of something like `getPathTail()` or `getFileFromPath()` ?